RNA-seq/README.md

```diff
@@ -15,3 +15,11 @@ The notebooks that comprise this module are (estimated time to complete notebook
 
 
 _Total estimated time to complete instruction notebooks: 3 hours 15 minutes_
+
+Beyond the notebooks mentioned above, this module has the following structure:
+
+![module project structure](./diagrams/bulk_project_structure.png)
```

Comment:
This figure isn't quite up to date, as there will be some changes when #376 goes in, notably to the leukemia directory and the addition of a medulloblastoma directory. Those might be best left collapsed though, as they might add confusion.